packages/flutter/lib/src/widgets/framework.dart

```diff
@@ -84,9 +84,18 @@ class ObjectKey extends Key {
 
 typedef void GlobalKeyRemoveListener(GlobalKey key);
 
-/// A GlobalKey is one that must be unique across the entire application. It is
-/// used by widgets that need to communicate with other widgets across the
-/// application's element tree.
+/// A GlobalKey is a [Key] that must be unique across the entire application.
```

Comment:
across the entire widget tree. "the application" is a bit vague and could be interpreted as meaning the lifetime of the application.